src/pages/Login/LoginPage.scss

```diff
@@ -1,59 +1,93 @@
 @import "../../theme";
+@import url(https://fonts.googleapis.com/css?family=PT+Sans);
 
-.login-page {
+.login-page{
     display: flex;
     flex-direction: row;
-    height: 100vh;
+    height: 100%;
+    width: 100%;
 
-    .left-column {
-
-        .landing-image {
-            width: 27vw;
-            height: 100%;
+    .left-container{
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+        width: 20vw;
+        
+        .andela-logo{
+            margin: 20% 0% 0% 10%;
+            width: 7.7vw;
+            height: 4.5vh;
         }
-    }
 
-    .right-column {
-        flex: 1;
+        .welcome-text{
+            margin-top: 40vh;
+            margin-left: 2vw;
+            width: 70%;
 
-        .login-container {
-            display: flex;
-            flex-direction: column;
-            align-items: center;
-            height: 100vh;
-            justify-content: center;
-            margin: auto !important;
 
-            .title {
-                margin-bottom: 0.3rem;
-                font-weight: 200;
-                font-size: 1.25rem;
-            }
+            p{
+                font-family: 'DIN Pro Extra Light';
+                font-size: 25px;
 
-            .instructions {
-                margin: 0.2rem;
+                .wire{
+                    font-family: 'DIN Pro Light';
+                    font-style: italic;
+                }
+                
             }
+        }
+        .button{
```

Comment:
Let's add a border radius to the button to resemble the mockup

Reply:
I tried that , it didnt work with the raised button component, is there another way or should be change it to a normal button ?